packages/google_maps_flutter/google_maps_flutter/ios/Classes/GoogleMapController.m

```diff
@@ -119,9 +113,25 @@ - (instancetype)initWithFrame:(CGRect)frame
 }
 
 - (UIView*)view {
+  [_mapView addObserver:self forKeyPath:@"frame" options:0 context:nil];
   return _mapView;
 }
 
+- (void)observeValueForKeyPath:(NSString*)keyPath
+                      ofObject:(id)object
+                        change:(NSDictionary*)change
+                       context:(void*)context {
+  if (object == _mapView && [keyPath isEqualToString:@"frame"]) {
+    [_mapView moveCamera:[GMSCameraUpdate setCamera:_mapView.camera]];
```

Comment:
What's the intended behavior if a the map frame is resized while it's being animated?

Reply:
In that case, this method will get called multiple times. However, since we are always setting the camera to the same location, it shouldn't have any behavior effects.

Reply:
[animateWithCameraUpdate](https://developers.google.com/maps/documentation/ios-sdk/reference/interface_g_m_s_map_view#a0a6b673be7ec23030c71484129aa9896) is implemented by multiple calls to `moveCamera`?

Reply:
I made this only happens once during initialization, so it shouldn't have any side effects that you worried about.